commcare_connect/reports/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from commcare_connect.users.models import User
+
+
+class UserAnalyticsData(models.Model):
+    username = models.CharField(unique=True)
+    # user can be null in cases where a user is only registered on PersonalID
+    # and is not a Connect User, the username field can be used as the
+    # user identifier in those cases.
+    user = models.OneToOneField(User, on_delete=models.CASCADE, null=True)
+
+    has_opp_invite = models.DateTimeField(null=True)
+    has_ever_earned_payment = models.DateTimeField(null=True)
+    has_started_learning = models.DateTimeField(null=True)
+    has_completed_learning = models.DateTimeField(null=True)
+    has_completed_assessment = models.DateTimeField(null=True)
+    has_claimed_job = models.DateTimeField(null=True)
+    has_started_job = models.DateTimeField(null=True)
+    has_paid = models.DateTimeField(null=True)
```

Comment:
this is "been paid" right? that wording might be confusing

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/824/commits/402768cd460404cbc439b90e8b254ea4c47b7193